tests/integration/cli/actions/test_auth_actions.py

```diff
@@ -39,79 +39,68 @@
-
-    # Setup
-    mock_click_prompt.return_value = INSECURE_DEVELOPMENT_PASSWORD
-    mock_click_confirm.return_value = True
-    test_envavr = 'NUCYPHER_TEST_ENVVAR'
+    mock_stdin.password(INSECURE_DEVELOPMENT_PASSWORD, confirm=confirm)
+    test_envvar = 'NUCYPHER_TEST_ENVVAR'
     another_password = 'th1s-iS-n0t-secur3'
 
-    mocker.patch.dict(os.environ, {test_envavr: another_password})
+    mocker.patch.dict(os.environ, {test_envvar: another_password})
     result = get_password_from_prompt(confirm=confirm)
     assert result == INSECURE_DEVELOPMENT_PASSWORD
-    mock_click_prompt.assert_called_once_with(GENERIC_PASSWORD_PROMPT,
-                                              confirmation_prompt=confirm,
-                                              hide_input=True)
-
-    mock_click_prompt.reset()
+    assert mock_stdin.empty()
+    captured = capsys.readouterr()
+    assert GENERIC_PASSWORD_PROMPT in captured.out
+    if confirm:
+        assert "Repeat for confirmation:" in captured.out
 
     # From env var
-    mocker.patch.dict(os.environ, {test_envavr: another_password})
-    result = get_password_from_prompt(confirm=confirm, envvar=test_envavr)
+    mocker.patch.dict(os.environ, {test_envvar: another_password})
+    result = get_password_from_prompt(confirm=confirm, envvar=test_envvar)
     assert result is not NO_PASSWORD
     assert result != INSECURE_DEVELOPMENT_PASSWORD
     assert result == another_password
-    mock_click_prompt.assert_called_once_with(GENERIC_PASSWORD_PROMPT,
-                                              confirmation_prompt=confirm,
-                                              hide_input=True)
+    assert mock_stdin.empty()
+    captured = capsys.readouterr()
+    assert not captured.out
+    assert not captured.err
 
 
-def test_get_client_password_with_invalid_address(mock_click_prompt, mock_account):
+def test_get_client_password_with_invalid_address(mock_stdin):
+    # `mock_stdin` used to assert the user was not prompted
     bad_address = '0xdeadbeef'
     with pytest.raises(InvalidChecksumAddress):
         get_client_password(checksum_address=bad_address)
 
 
 @pytest.mark.parametrize('confirm', (True, False))
-def test_get_client_password(mock_click_prompt, mock_account, confirm):
-    mock_click_prompt.return_value = INSECURE_DEVELOPMENT_PASSWORD
+def test_get_client_password(mock_stdin, mock_account, confirm, capsys):
+    mock_stdin.password(INSECURE_DEVELOPMENT_PASSWORD, confirm=confirm)
     result = get_client_password(checksum_address=mock_account.address, confirm=confirm)
     assert result == INSECURE_DEVELOPMENT_PASSWORD
+    assert mock_stdin.empty()
     message = COLLECT_ETH_PASSWORD.format(checksum_address=mock_account.address)
-    mock_click_prompt.assert_called_once_with(message, confirmation_prompt=confirm, hide_input=True)
+    captured = capsys.readouterr()
+    assert message in captured.out
+    if confirm:
+        assert "Repeat for confirmation:" in captured.out
```

Comment:
I wonder if click provides an exportable name for this string.

Reply:
I doubt it: https://github.com/pallets/click/blob/97de2c97275f52adc5c6ef8bb8aa993ce997fe7e/src/click/termui.py#L161